From acbf3e7a1b01de8da1f57405e4a27d8c3fa6cc89 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 11 Jun 2023 19:39:57 +0200
Subject: [PATCH 298/464] arm64: dts: rockchip: rk3399-pinephone-pro: Add I2C
 supply to stk3311

It is needed to communicate with the device, when other users of
vcc3v0_touch shut down the regulator.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index f9d85a40ab13..a82232e29f93 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -848,6 +848,7 @@ light-sensor@48 {
 		pinctrl-0 = <&stk3311_int>;
 		vdd-supply = <&vcc_3v0>;
 		leda-supply = <&vcc_3v0>;
+		i2c-supply = <&vcc3v0_touch>;
 	};
 };
 
-- 
2.34.1

